js/core/telemetryV3Interface.js

```diff
@@ -127,7 +127,13 @@ var Telemetry = (function() {
      */
     this.telemetry.assess = function(data, options) {
         instance.updateValues(options);
-        instance._dispatch(instance.getEvent('ASSESS', data));
+        assessEvent = instance.getEvent('ASSESS', data);
+        // This code will replace current version with the new version number, if present in the data.
+        if (data.item && data.item.eventVer) {
+            assessEvent.ver = data.item.eventVer;
```

Comment:
Send "eventVer" in the "options" parameter instead of "data" parameter

Reply:
Now sending eventVer in options field.